src/switch_exporter/switch.py

```diff
@@ -13,7 +13,7 @@
 
 
 logger = logging.getLogger(__name__)
-_PORT_RE = re.compile(r'^Eth([^:]*):?$')
+_PORT_RE = re.compile(r'^Eth([^ :]*)(?: \(.*\))?:?$')
```

Comment:
The second capture group which you're just discarding here isn't just an arbitrary set of characters, it'll be something like `\(Po[\d]+\)`.

I'm not sure if this information will be useful, if we'd like to consider the LAGs separately for instance.

I'm not currently aware of any circumstances where anything other than `Po[digit]` will be there so I am not sure of the value of trying to be more specific.

Reply:
I was also unsure about how specific to try to be. Too specific and we're just going to have to revisit when some other string turns up; too generic and we might mis-identify something. Without any examples of what else might occur it's hard to make an informed decision, but my gut feel is that if something else appears between parentheses, we probably want to ignore it too.